jfrog-cli/artifactory/commands/config.go

```diff
@@ -315,9 +341,9 @@ func EncryptPassword(details *config.ArtifactoryDetails) (*config.ArtifactoryDet
 }
 
 func checkSingleAuthMethod(details *config.ArtifactoryDetails) error {
-	boolArr := []bool{details.User != "" && details.Password != "", details.ApiKey != "", fileutils.IsSshUrl(details.Url)}
+	boolArr := []bool{details.User != "" && details.Password != "", details.ApiKey != "", fileutils.IsSshUrl(details.Url), details.AccessToken != ""}
 	if cliutils.SumTrueValues(boolArr) > 1 {
-		return errorutils.CheckError(errors.New("Only one authentication method is allowd: Username/Password, API key or RSA tokens."))
+		return errorutils.CheckError(errors.New("Only one authentication method is allowd: Username/Password, API key, RSA tokens or access tokens"))
```

Comment:
Let's rephrase a bit to:
Only one authentication method is allowed: Username + Password/API key, RSA Token (SSH) or Access Token.